configs/sagan/README.md

```diff
@@ -0,0 +1,59 @@
+
+'\*' Iteration counting rule in our implementation is different from others. If you want to align with other codebases, you can use the following conversion formula:
+```
+Total_iters* = Total_iter / (n_disc * n_grad_accu)
+```
+
+We also provide converted pretrain model from [Pytorch-StudioGAN](https://github.com/POSTECH-CVLab/PyTorch-StudioGAN).
+To be noted that, in Pytorch Studio GAN, **inplace ReLU** is used in generator and discriminator.
+
+|  Models   | Dataset  | Inplace ReLU | n_disc | n_grad_accu | Total Iters |  IS\*  |  FID\*  | IS\*\* | FID\*\* |                                                          Download                                                           |                                Original Download link                                |
+| :-------: | :------: | :----------: | :----: | :---------: | :---------: | :----: | :-----: | :----: | :-----: | :-------------------------------------------------------------------------------------------------------------------------: | :----------------------------------------------------------------------------------: |
+| sagan_32  | CIFAR10  |      w       |   5    |      1      |   100000    | 9.116  | 10.2011 | 8.680  | 14.009  |   [Download](https://download.openmmlab.com/mmgen/sagan/sagan_32_cifar10_convert-studio-rgb_20210730_153321-080da7e2.pth)   | [Download](https://drive.google.com/drive/folders/1FA8hcz4MB8-hgTwLuDA0ZUfr8slud5P_) |
+| sagan_128 | ImageNet |      w       |   1    |      1      |   1000000   | 27.367 | 40.1162 | 29.848 | 34.726  | [Download](https://download.openmmlab.com/mmgen/sagan/sagan_128_imagenet1k_convert-studio-rgb_20210730_153357-eddb0d1d.pth) | [Download](https://drive.google.com/drive/folders/1ZYaqeeumDgxOPDhRR5QLeLFIpgBJ9S6B) |
+
+
+* '\*\*' denote results evaluated with our pipeline.
+* '\*\*\*' denote results released by Pytorch-StudioGAN.
+
+For IS evaluation,
+1. We use [Tero's Inception](https://nvlabs-fi-cdn.nvidia.com/stylegan2-ada-pytorch/pretrained/metrics/inception-2015-12-05.pt) for feature extraction.
+2. We use bicubic interpolation with PIL backend to resize image before feed them to Inception.
+
+For FID evaluation, difference between two repositories mainly on the selection of images for ground truth activation (inception state) calculation. In ours, we follow the pipeline of BigGAN for more convincing results. Detailly, we use the entire training set (50k and 1281167 images for CIFAR10 and ImageNet1k) without shuffle for GT inception state extracting. We also use [Tero's Inception](https://nvlabs-fi-cdn.nvidia.com/stylegan2-ada-pytorch/pretrained/metrics/inception-2015-12-05.pt) for feature extraction.
```

Comment:
For FID evaluation, differences between PyTorch Studio GAN and ours are mainly on the selection of real samples. In MMGen, we follow the pipeline of BigGAN, where the whole training set is adopted to extract inception statistics. Besides, we also use [Tero's Inception](https://nvlabs-fi-cdn.nvidia.com/stylegan2-ada-pytorch/pretrained/metrics/inception-2015-12-05.pt) for feature extraction.

Reply:
**are** mainly on

Reply:
It seems that you have not modified the sentences.